docs/vs-tools-tizen/tools/emulator-manager.md

```diff
@@ -1,148 +1,141 @@
 
 There are 2 different ways to access the Emulator Manager from Visual Studio:
 
-* Go to ```Tools > Tizen```, and select ```Tizen Emulator Manager```
-
-   ![Launch the Emulator Manager in the Tools menu](../image/em_vstoolbar_1.png)
+- In the Visual Studio menu, go to **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
 
-* Click ```Launch Tizen Emulator``` on the Visual Studio toolbar.
+   ![Launch the Emulator Manager in the Tools menu](media/em-vstoolbar1.png)
 
-   ![Launch the Emulator Manager in the Visual Studio](../image/em_vstoolbar_2.png)
+- On the Visual Studio toolbar, click **Launch Tizen Emulator**.
 
----
+   ![Launch the Emulator Manager in the Visual Studio](media/em-vstoolbar2.png)
 
-### Creating an Emulator Instance ###
 
-The Emulator Manager can help you to select the recommended platform and template. When you need another device environment, you can edit the existing emulator instance, or create a new one with a more suitable platform and template. You can also create platforms and templates to suit your needs.
+## Creating Emulator Instances
 
-![Emulator Manager](../image/em_vs_1.png)
+The Emulator Manager can help you to select the recommended platform and template. When you need another device environment, you can edit the existing emulator instance, or create a new one with a more suitable platform and template. You can also create new platforms and templates to suit your needs.
 
-To create a new emulator instance:
+**Figure: Emulator Manager**
 
-1. In the Emulator Manager, click ```Create```.
-2. Select the platform (system image), and click ```Next```.
-3. Select the template (device definition), and click ```Next```.
-4. Change properties as needed, and click ```Finish```.
+![Emulator Manager](media/em-vs1.png)
 
-The emulator instance appears in the Emulator Manager.
-
-To view the detailed information, right-click the emulator to see a context menu.
+To create a new emulator instance:
 
-```
-Note
+1. In the Emulator Manager, click **Create**.
+2. Select the platform (system image), and click **Next**.
+3. Select the template (device definition), and click **Next**.
+4. Change properties as needed, and click **Finish**.
 
-To run the application faster, switch on CPU VT and GPU. If CPU VT is disabled, check Increasing the Application Execution Speed for more information. If GPU is disabled, install the latest vendor-provided graphic driver.
+The emulator instance appears in the Emulator Manager. To view the emulator instance details, right-click the emulator and select **Detail**.
 
-Mobile emulator supports HD(720x1280) or WVGA(480x800) resolutions.
-```
+> **Note**
+>
+> To run the application faster, switch on CPU VT and GPU. If CPU VT is disabled, check Increasing the Application Execution Speed for more information. If GPU is disabled, install the latest vendor-provided graphic driver.
```

Comment:
“If CPU VT is disabled, check Increasing the Application Execution Speed for more information.” is missing a link. Where should the link lead, or should this sentence be removed?

Reply:
Link added